site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,260 +1,252 @@
 ---
-title: 'Creating an annotation task'
-linkTitle: 'Creating an annotation task'
+title: 'Create annotation task'
+linkTitle: 'Create annotation task'
 weight: 2
-description: 'Instructions on how to create and configure an annotation task.'
+description: 'How to create and configure an annotation task.'
 ---
 
-Create an annotation task pressing `+` button and select `Create new task` on the tasks page or on the project page.
+To start annotating in CVAT, you need to create an annotation task and specify its parameters.
 
-![](/images/image004.jpg)
+To create a task, on the **Tasks** or **Projects** page click **+** and
```

Comment:
You can create a task for a specific project on a Project page. But I believe we don't have the option to create a task on the Projects page. Am I right?